scripts/send-message-from-rialto-millau.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Used for manually sending a message to a running network.
+#
+# You could for example spin up a full network using the Docker Compose files
+# we have (to make sure the message relays are running), but remove the message
+# generator service. From there you may submit messages manually using this script.
+
+case "$1" in
+	remark)
+		RUST_LOG=runtime=trace,substrate-relay=trace,bridge=trace \
+		./target/debug/substrate-relay send-message rialto-to-millau \
+			--rialto-host localhost \
+			--rialto-port 9944 \
+			--millau-signer //Alice \
+			--rialto-signer //Bob \
+			--lane 00000000 \
+			--origin Target \
+			remark \
+		;;
+	transfer)
+		RUST_LOG=runtime=trace,substrate-relay=trace,bridge=trace \
+		./target/debug/substrate-relay send-message rialto-to-millau \
+			--rialto-host localhost \
+			--rialto-port 9944 \
+			--millau-signer //Alice \
+			--rialto-signer //Bob \
+			--lane 00000000 \
+			--origin Target \
+			transfer \
+			--amount 100000000000000 \
+			--recipient 5DZvVvd1udr61vL7Xks17TFQ4fi9NiagYLaBobnbPCP14ewA \
```

Comment:
iirc this used to be an address you could see on the UI. Given the recent address encoding changes I think this will need to be changed in both `send-message` scripts

Reply:
Actually after #781 CLI works with addresses encode with any SS58 prefix, it will only display a warning in case it doesn't match the network.